### DISCUSSION TITLE
Add a random option when letting songs play on homescreen

### DISCUSSION THREAD

**Discussion by notTPALT** [ppy/osu#23118] [Ideas] [answered]
I've been using osu as a media player for like 2 years. I notice that unlike stable, lazer play songs in 'date added' order, and there's no option for it. Please make it random, I'm tired of scrolling and select one by one.

Sorry for my bad English.

**Comment by Rekunan** [comment]
Likely won't be implemented as osu! isn't meant to be a media player.

**Reply by notTPALT** [reply]
I know, but since they made the 'now playing' tab this good
![image](https://user-images.githubusercontent.com/92924640/229544838-09a8ef48-19c8-43db-9825-d59f722d8bc5.png)
It is quite a miss that it's not playing on random.

**Comment by peppy** [comment, marked as answer]
https://github.com/ppy/osu/issues/18169
